yarn-project/aztec-nr/aztec/src/note/note_getter.nr

```diff
@@ -143,24 +125,18 @@ unconstrained fn get_notes_internal<Note, N, FILTER_ARGS>(
         options.limit,
         options.offset,
         placeholder_opt_notes,
-        placeholder_fields,
-    );
+        placeholder_fields);
 
     let filter = options.filter;
     let filter_args = options.filter_args;
     filter(opt_notes, filter_args)
 }
 
-unconstrained pub fn view_notes<Note, N>(
-    storage_slot: Field,
-    note_interface: NoteInterface<Note, N>,
-    options: NoteViewerOptions<Note, N>,
-) -> [Option<Note>; MAX_NOTES_PER_PAGE] {
+unconstrained pub fn view_notes<Note, N>(storage_slot: Field, note_interface: NoteInterface<Note, N>, options: NoteViewerOptions<Note, N>) -> [Option<Note>; MAX_NOTES_PER_PAGE] {
```

Comment:
@f01dab1e I know you are most likely aware of this but having line length set to 120 by default would be useful. Is max line length already supported?

Reply:
Yes, this can be done. In noirfmt.toml:
```toml
max_width = 120
```
The list of settings is here: https://github.com/noir-lang/noir/blob/master/tooling/nargo_fmt/src/config.rs#L44-L53